File: reverse-string/reverse-string.js
/**
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
var reverseString = function(s) {
    let left = 0;
    let right = s.length -1;
    
    while (left < right) {
        let x = s[left]
        s[left] = s[right]
        s[right] = x
        left +=1
        right -=1

    }
    return s
};

File: reverse-string/reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        n = len(s)
        i = 0
        j = n - 1
        
        while i < j:
            dummy = s[i]
            dummyTwo = s[j]
            s[i] = dummyTwo
            s[j] = dummy
            i += 1
            j -= 1
     
        
        return s
        